test/integration/puppetlabs/services/master/environment_classes_int_test.clj

```diff
@@ -92,9 +94,47 @@
   [response]
   (-> response :body cheshire/parse-string))
 
-(deftest ^:integration environment-classes-integration-test
+(deftest ^:integration environment-classes-integration-cache-disabled-test
+  (testing "when environment classes cache is disabled for a class request"
+    (bootstrap/with-puppetserver-running-with-config
+     app
+     (-> {:jruby-puppet {:max-active-instances 1}}
+         (bootstrap/load-dev-config-with-overrides)
+         (ks/dissoc-in [:puppetserver
+                        :environment-class-cache-enabled]))
```

Comment:
I found this line really confusing and didn't understand why it was here until I traced back what `load-dev-config-with-overrides` did... which is copy over the dev sample config file. Since the default setting for this is `false` I feel like it might be less confusing to have the sample config file set it to false, or not set it at all.